docs/tutorials/settings_tutorial.ipynb

```diff
@@ -0,0 +1,437 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Setup\n",
+    "\n",
+    "This tutorial aims to show how to use the `Settings` class to configure `PaperQA`.\n",
+    "Firstly, we will be using `OpenAI` and `Anthropic` models, so we need to set the `OPENAI_API_KEY` and `ANTHROPIC_API_KEY` environment variables.\n",
+    "We will use both models to make it clear when `paperqa` agent is using either one or the other.\n",
+    "We use `python-dotenv` to load the environment variables from a `.env` file. \n",
+    "Hence, our first step is to create a `.env` file and install the required packages."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!echo \"OPENAI_API_KEY=<your-openai-api-key>\" > .env\n",
+    "!echo \"ANTHROPIC_API_KEY=<your-anthropic-api-key>\" >> .env\n",
+    "\n",
+    "!uv pip install -q nest-asyncio python-dotenv aiohttp fhlmi \"paper-qa[local]\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "import aiohttp\n",
+    "import nest_asyncio\n",
+    "from dotenv import load_dotenv\n",
+    "\n",
+    "nest_asyncio.apply()\n",
+    "load_dotenv(\".env\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(\"You have set the following environment variables:\")\n",
+    "print(f\"OPENAI_API_KEY: {os.environ['OPENAI_API_KEY']}\")\n",
+    "print(f\"ANTHROPIC_API_KEY: {os.environ['ANTHROPIC_API_KEY']}\")"
```

Comment:
I feel like someone out there is going to run this, forget to clear outputs, share the notebook, and thus leak these secrets.

What do you think of removing this cell?